### PR TITLE
LPS-178164 revome FF

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
https://issues.liferay.com/browse/LPS-178164

- LPS-178164 remove FF from UserNotificationResource impl and test
- LPS-178164 remove FF from poshi test
- LPS-178164 buildRest

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:headless

**Comment by AliciaGarciaGarcia** [comment]
@ccorreagg can you take a look?

File: modules/apps/headless/headless-user-notification/headless-user-notification-test/src/testIntegration/java/com/liferay/headless/user/notification/resource/v1_0/test/UserNotificationResourceTest.java [hunk 1]
```diff
@@ -47,22 +45,12 @@ public void setUp() throws Exception {
...
-
-		PropsUtil.addProperties(
-			UnicodePropertiesBuilder.setProperty(
-				"feature.flag.LPS-83384", "true"
-			).build());
 	}
 
 	@After
 	@Override
 	public void tearDown() throws Exception {
```
Comment (ccorreagg) [inline modules/apps/headless/headless-user-notification/headless-user-notification-test/src/testIntegration/java/com/liferay/headless/user/notification/resource/v1_0/test/UserNotificationResourceTest.java:52]: This method might be removed

**Comment by ccorreagg** [comment]
The FF has to be removed also from the portal.properties file

**Comment by AliciaGarciaGarcia** [comment]
@ccorreagg 
Seems that the test failures only happened with my machine. 🤷🏻‍♀️  
I uploaded the SF and the change on the test.

Thanks for review it

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:headless

**Review by ccorreagg — APPROVED** [review]
LGTM now!

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
@AliciaGarciaGarcia Everything looks good. I'm not sure if this can be forwarded, so I'll let you do the honors.

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by WYZ-Issac** [comment]
ci:forward:force

**Pull request closed without merging**